lib/stats/stat.js

```diff
@@ -12,7 +15,14 @@ export default class Stat {
     assert(config.workerNodeType, 'Worker instance type is required');
     assert(config.provisionerId, 'Provisioner ID is required');
 
-    this.influx = new base.stats.Influx(config.influx);
+    if (config.influx.connectionString) {
```

Comment:
I guess this is okay... but really can't we just make a hack in `base.stats` to support doing this? Maybe if connectionString = `null` then we print with debug?